src/robusta/core/reporting/base.py

```diff
@@ -272,18 +272,21 @@ def __str__(self):
 
     def get_prometheus_silence_url(self, account_id: str, cluster_name: str) -> str:
         labels: Dict[str, str] = {"alertname": self.aggregation_key, "cluster": cluster_name, "account": account_id}
-        if self.subject.namespace:
-            labels["namespace"] = self.subject.namespace
+        kind: Optional[str] = self.subject.subject_type.value
+        if kind and kind == "node":
+            pass
+        else:
+            if self.subject.namespace:
```

Comment:
couldnt 
```
        if kind and kind == "node":
            pass
        else:
            if self.subject.namespace:
```
    be simplified as 
`if kind != "node" and self.subject.namespace:`